src/dispatch/auth/views.py

```diff
@@ -163,6 +166,9 @@ def update_user(
         UserOrganization(role=user_in.role, organization=OrganizationRead(name=organization))
     ]
 
+    # we currently only allow user password changes via CLI, not UI/API.
+    user_in.password = None
```

Comment:
This will make this [if](https://github.com/Netflix/dispatch/blob/master/src/dispatch/auth/service.py#L223) to evaluate as `False`, so password won't be updated via API, but still allow for password changes via CLI.